contracts/router/AedexV2Router.aes

```diff
@@ -93,21 +97,25 @@ payable contract AedexV2Router =
                 amount_a_min, amount_b_min,
                 min_liquidity
                 )
-        //getting pair and transfer to its address the tokens
+        //getting pair and transfer the tokens to its address
         let pair: IAedexV2Pair = AedexV2Library.pair_for( state.factory, token_a, token_b )
         token_a.transfer_allowance( Call.caller,pair.address,amount_a )
         token_b.transfer_allowance( Call.caller,pair.address,amount_b )
         let minted_liquidity = pair.mint(to)
         ( amount_a, amount_b, minted_liquidity )
 
+    // Adds liquidity for a certain Pair of a regular AEX9 token and AE.
+    // If the Pair doesn't exists the pair will be created at this moment
+    // and the min_liquidity will be locked
     payable stateful entrypoint add_liquidity_ae(
         token: IAEX9Minimal
         , amount_token_desired: int
         , amount_token_min: int
         , amount_ae_min: int
         , to: address
-        , min_liquidity: int // the minimum liquidity constant for the pair
-                             // in case the pair is not yet created
+        , min_liquidity: option(int) // the minimum liquidity constant for the pair
```

Comment:
Same as above comment, this should be understood if you only refer to the documentation of `add_liquidity` and mention how this is different instead of duplicating the comment.